src/fmu/dataio/hook_implementations/jobs.py

```diff
@@ -1,8 +1,10 @@
-from ert_shared.plugins.plugin_manager import hook_implementation
-from ert_shared.plugins.plugin_response import plugin_response
+try:
+    import ert_shared
+except ModuleNotFoundError:
+    import ert.shared as ert_shared
```

Comment:
Could not make one suggestion for some reason
```suggestion
try:
    from ert_shared.plugins.plugin_manager import hook_implementation, plugin_response  # type: ignore
except ModuleNotFoundError:
    from ert.shared.plugins.plugin_manager import hook_implementation, plugin_response # type: ignore
```